Update readme (#58)
instamatic-dev/instamatic: readme.md

Alternatively, download the portable installation with all libraries/dependencies included: https://github.com/instamatic-dev/instamatic/releases/latest. Extract the archive, and open a terminal by double-clicking `start_Cmder.exe`.

Installation should take less than 1 minute on a normal desktop.

Find the latest [releases](https://github.com/instamatic-dev/instamatic/releases) for the versions that have been tested on.

## OS requirement

> The package has been mainly developed and tested under windows 7 and windows 10.

Author: ```suggestion
The package requires Windows 7 or higher. It has been mainly developed and tested under windows 7 and windows 10.
```

## Package dependencies

> Check [requirements.txt](https://github.com/instamatic-dev/instamatic/blob/master/requirements.txt) for an updated package dependency list and versions.

Author: ## Package dependencies

Check [pyproject.toml](pyproject.toml) for the full dependency list and versions.

> ## Package dependencies
> 
> Check [requirements.txt](https://github.com/instamatic-dev/instamatic/blob/master/requirements.txt) for an updated package dependency list and versions.

Author: ```suggestion
## Package dependencies

Check [pyproject.toml](pyproject.toml) for the full dependency list and versions.
```
